content/doc/pipeline/tour/getting-started.adoc

```diff
@@ -29,7 +29,13 @@ For this tour, you will require:
 . Follow the instructions to complete the installation.
 
 When the installation is complete, you can start putting Jenkins to work!
+[NOTE]
+====
+There is a https://issues.jenkins-ci.org/browse/INFRA-1659[bug] in the current and older Jenkins builds when using Docker. The Jenkins certificate uses 1024-bit key. This has been deprecated in some Linux distrobutions, like https://fedoraproject.org/wiki/Changes/StrongCryptoSettings[Fedora 28+], because they accept at least 2048-bit keys. There is a way to circumvent the security check by lowering the accepted key size, but it is recommended to install Jenkins link:/doc/book/installing/index.adoc[locally] for now instead.
```

Comment:
```suggestion
There is a https://issues.jenkins-ci.org/browse/INFRA-1659[bug] in the current and older Jenkins builds when using Docker. 
The Jenkins certificate uses 1024-bit key. 
This has been deprecated in some Linux distributions, such as https://fedoraproject.org/wiki/Changes/StrongCryptoSettings[Fedora 28+], because they accept at least 2048-bit keys. 
There is a way to circumvent the security check by lowering the accepted key size, but it is recommended to install Jenkins link:/doc/book/installing/index.adoc[locally (outside of Docker] for now instead.

```